ecl/hql/hqlexpr.hpp

```diff
@@ -1022,6 +1022,7 @@ class HQL_API HqlLookupContext
     inline bool syntaxChecking() const { return parseCtx.isSyntaxChecking(); }
     inline bool regenerateCache() const { return parseCtx.regenerateCache; }
     inline bool hasCacheLocation() const { return parseCtx.hasCacheLocation();}
+    inline bool syntaxCheckingUsingCache() const { return parseCtx.isSyntaxChecking() && parseCtx.hasCacheLocation(); }
```

Comment:
discussion: I would suggest calling this suppressSemanticErrors().  The reason is that it is then self documenting at the point of call.  if (!lookupCtx.suppressSemantciErrors() || ...)
The reason for raining it is that I'm not sure if this condition is quite right.  For instance when running the regression suite to test the syntax checking you also need to suppress the errors.  I think it would currently be !isSyntaxchecking && !ignoreCache.

Reply:
I've removed this function because after commoning up the code below this function is used just once.